content/applications/inventory_and_mrp/inventory/shipping/operation/multipack.rst

```diff
@@ -1,84 +1,119 @@
-=========================================================
-How to manage multiple packs for the same delivery order?
-=========================================================
+=======================
+Multi-package shipments
+=======================
 
-Overview
-========
-
-By default, Odoo considers that your delivery is composed of one
-package. But you can easily ship your deliveries with more than one
-package. It is fully integrated with the third-party shippers.
+In some cases, a delivery order with multiple items may need to be shipped in more than one package.
+This may be necessary if the items are too large to ship in a single package, or if certain items
+cannot be packaged together. Shipping a single delivery order in multiple packages provides
+flexibility for how each item is packaged, without the need to create multiple delivery orders.
 
 Configuration
 =============
 
-To configure the use of packages, go to the menu
-:menuselection:`Inventory --> Configuration --> Settings`.
-Locate the **Packages** section and tick **Record
-packages used on packing: pallets, boxes,...**
+In order to split a delivery order across multiple packages, the *Packages* setting must be enabled.
+To do so, navigate to :menuselection:`Inventory --> Configuration --> Settings`, then enable the
+checkbox next to :guilabel:`Packages`. Click :guilabel:`Save` to confirm the change.
 
-.. image:: multipack/multipack03.png
+.. image:: multipack/packages-setting.png
    :align: center
+   :alt: The Packages setting on the Inventory app settings page.
 
-Click on **Apply** when you are done.
+Ship items in multiple packages
+===============================
 
-Sale process
-============
+To split items in the same delivery order across multiple packages, begin by navigating to
+:menuselection:`Inventory --> Delivery Orders`, then select a delivery order that has multiple
+items, a multiple quantity of the same item, or both.
 
-Sale order
-----------
+On the :guilabel:`Operations` tab, select the :guilabel:`⁞≣` menu icon in the line of the product
```

Comment:
Let's put the icon descriptor within the guilabel in parenthesis. Also, I like how you combined two icons to create the exact icon in the UI rather than just "≣" 👍
```suggestion
On the :guilabel:`Operations` tab, select the :guilabel:`⁞≣ (menu)` icon in the line of the product
```